ci.sh

```diff
@@ -15,4 +15,14 @@ npm install -g pxt
 )
 
 npm link ../pxt
+
+# Add ml extension
+cd libs
+git clone -b v0.1.23 git@github.com:microbit-foundation/pxt-ml-extension-poc.git
```

Comment:
Latest build-able version is v0.1.23. The latest version is not build-able as it is being worked on.